cmd/osbuild-tests/main_test.go

```diff
@@ -77,14 +77,18 @@ func TestComposeCommands(t *testing.T) {
 	require.NoError(t, err, "'%s-disk.qcow2' not found", uuid.String())
 	defer os.Remove(uuid.String() + "-disk.qcow2")
 
+	// workers ask the composer every 15 seconds if a compose was canceled.
+	// Use 20 seconds here to be sure this is hit.
+	uuid = startCompose(t, "empty", "qcow2")
+	defer deleteCompose(t, uuid)
+	runComposer(t, "compose", "cancel", uuid.String())
+	time.Sleep(20 * time.Second)
+	status := waitForCompose(t, uuid)
+	assert.Equal(t, "FAILED", status)
```

Comment:
Isn't there a status CANCELLED to distinguish when the user cancelled the compose or is all FAILED ?

Reply:
Alas, there is not.